### PR TITLE
docs: clarify steps in Hosting on Firebase

### PR DESCRIPTION
**Pull request by DavSanchez**
Hi guys!

I have added some clarifications on the `firebase init` process and a small correction on the `firebase deploy` line, at the Deployment section of the Getting Started docs.

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by DavSanchez** [comment]
@googlebot I signed it!

**Comment by kapunahelewong** [comment]
Thanks for this, @DavSanchez! I've been working through these myself and found that that part definitely needed some more detail. Could you rebase and amend your commit message to be:

> docs: clarify steps in Hosting on Firebase

In case you'd like them, I like to share my own workflow with others. Here are [my notes on rebasing](https://gist.github.com/kapunahelewong/567c30f7b353fd1c39bbf6d8343e2dca#how-to-rebase) and [my notes on amending commit messages](https://gist.github.com/kapunahelewong/567c30f7b353fd1c39bbf6d8343e2dca#how-to-amend-a-commit).

Please feel free to tag me with any questions and when you're ready. Thank you!

**Comment by DavSanchez** [comment]
Hi @kapunahelewong, thanks for the links! I'll do the rebase and amend and come back to you with any questions I might have.

**Comment by DavSanchez** [comment]
Alright! I think I did it correctly, although I ended up amending first and rebasing after, hence the two commits... is that a problem?

File: aio/content/start/deployment.md [hunk 1]
```diff
@@ -68,8 +68,11 @@ One of the easiest ways to get your site live is to host it using Firebase.
 1. Create a new project, giving it any name you like.
 1. Install the `firebase-tools` CLI that will handle your deployment using `npm install -g firebase-tools`.
 1. Connect your CLI to your Firebase account and initialize the connection to your project using `firebase login` and `firebase init`.
-1. Follow the prompts to select the `Firebase` project you creating for hosting.
-1. Deploy your application with `firebase deploy` because StackBlitz has created a `firebase.json` that tells Firebase how to serve your app.
+1. Follow the prompts to select the `Firebase` project you are creating for hosting
```
Comment (kapunahelewong) [inline aio/content/start/deployment.md]: Oops, I missed this before. Can you put a period at the end of the sentence?
```suggestion
1. Follow the prompts to select the `Firebase` project you are creating for hosting.
```

**Review by kapunahelewong — CHANGES_REQUESTED** [review]
It looks like you did a perfect job with the rebase/squashing, @DavSanchez. 🌟I just realized though that I missed one little thing yesterday. 

Regarding rebasing before or after squashing, it doesn't really matter if the change is small. Sometimes, though, with a PR with lots of changes, if you don't squash first it can get a little repetitive because the rebase goes commit by commit. So first git will rebase your first commit, then the second and so on. So, if you have any conflicts, it's possible to have some of the same ones over and over again. I found this out with a branch I had with about 10 commits once!

Anyway, when you're ready (sorry for my oversight), will you tag me again? I'll keep an eye out for you. 🙂

File: aio/content/start/deployment.md (hunk 1 above)
Comment (DavSanchez) [reply aio/content/start/deployment.md]: Yes! Already committed @kapunahelewong . If you need anything else please don't hesitate to let me know!

**Comment by kapunahelewong** [comment]
Ah, I haven't experimented with using the UI for committing changes yet. Yes, you should be able to fix it with an amend, but really, since you want to only have 1 commit in your PR, if you squash, you can just fixup that last one and be in good shape. Here's the [squashing section of my notes](https://gist.github.com/kapunahelewong/567c30f7b353fd1c39bbf6d8343e2dca#how-to-squash). Let me know if you have any questions. :)

**Comment by DavSanchez** [comment]
Alright @kapunahelewong! I think it's already done. If you need anything else please let me know!

**Review by kapunahelewong — APPROVED** [review]
LGTM! Thank you very much, @DavSanchez!

Caretaker, this is a docs only change. Thank you!

**Pull request closed without merging**